### PR TITLE
fixed broken link

### LINKED ISSUE #465: [BUG]: link not working
### Description

At the end of the overview (documentation) page there's a "edit this page" link, on clicking the link a 404 error page pops up. 

### Screenshots

image
image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by barkhaaroraa**
fixes #465 currently taking the user to the github repo to make changes

### CODE REVIEW

**Comment by barkhaaroraa** [comment]
@origranot could you please review and merge this

**Comment by origranot** [comment]
@barkhaaroraa Hey,
I think it should redirect to the specific .MD file inside the docs app.

File: apps/docs/docusaurus.config.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const config = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           routeBasePath: '/',
-          editUrl: '/',
+          editUrl: 'https://github.com/origranot/reduced.to',
```
Comment (origranot) [inline apps/docs/docusaurus.config.js:27]: Hey @barkhaaroraa,
editUrl is wrong, take a look at: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs

**Review by origranot — CHANGES_REQUESTED** [review]
@barkhaaroraa
It looks like the editUrl is wrong, please test it locally before opening a PR.

**Pull request closed without merging**